docs/SampleApplications-Movies.md

```diff
@@ -1,71 +1,84 @@
+    project in `SampleAppMovies/iOS/SampleAppMovies.xcodeproj` and a gradle project in
+    `SampleAppMovies/android/app`.
 
 ### Starting the app on iOS
 
-Open this new project (`AwesomeProject/ios/AwesomeProject.xcodeproj`) in Xcode and simply build and run it with `⌘+R`. Doing so will also start a Node server which enables live code reloading. With this you can see your changes by pressing `⌘+R` in the simulator rather than recompiling in Xcode.
+Open this new project (`SampleAppMovies/ios/SampleAppMovies.xcodeproj`) in Xcode and simply build
+and run it with `⌘+R`. Doing so will also start a Node server which enables live code reloading.
+With this you can see your changes by pressing `⌘+R` in the simulator rather than recompiling in
+Xcode.
 
 ### Starting the app on Android
 
-In your terminal navigate into the `AwesomeProject` and run:
+In your terminal navigate into the `SampleAppMovies` and run:
 
     react-native run-android
 
-This will install the generated app on your emulator or device, as well as start the Node server which enables live code reloading. To see your changes you have to open the rage-shake-menu (either shake the device or press the menu button on devices, press F2 or Page Up for emulator, ⌘+M for Genymotion), and then press `Reload JS`.
+This will install the generated app on your emulator or device, as well as start the Node server
+which enables live code reloading. To see your changes you have to open the rage-shake-menu (either
+shake the device or press the menu button on devices, press F2 or Page Up for emulator, ⌘+M for
+Genymotion), and then press `Reload JS`.
 
 ### Hello World
 
-`react-native init` will generate an app with the name of your project, in this case AwesomeProject. This is a simple hello world app. For iOS, you can edit `index.ios.js` to make changes to the app and then press ⌘+R in the simulator to see the changes. For Android, you can edit `index.android.js` to make changes to the app and press `Reload JS` from the rage shake menu to see the changes.
+`react-native init` will generate an app with the name of your project, in this case
+`SampleAppMovies`. This is a simple hello world app. For iOS, you can edit `index.ios.js` to make
+changes to the app and then press ⌘+R in the simulator to see the changes. For Android, you can
+edit `index.android.js` to make changes to the app and press `Reload JS` from the rage shake menu
+to see the changes.
+
+## Actual App
+
+Now that we have initialized our React Native project, we can begin creating our Movie application.
 
 ### Mocking data
 
-Before we write the code to fetch actual Rotten Tomatoes data let's mock some data so we can get our hands dirty with React Native. At Facebook we typically declare constants at the top of JS files, just below the imports, but feel free to add the following constant wherever you like. In `index.ios.js` or `index.android.js` :
```

Comment:
Would be nice to not wrap at 80 columns. It makes it a lot more annoying to edit text as if you edit the first line of the paragraph, you need to change the 3 lines afterwards in order to make sure they wrap correctly. In every text editor there's a setting to wrap the lines to fit the size of your editor so that you don't edit one giant line.